src/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -154,7 +155,24 @@ private function getModuleExecutionConfig()
             ->enumNode('strategy')
                 ->values(['inline', 'subrequest'])
                 ->defaultValue('inline')
-                ->info('Set to "subrequest" to execute modules as Symfony subrequests; set to "inline" to abstain from subrequests and improve performance')
+                ->info('Set to "subrequest" to execute modules as Symfony subrequests; set to "inline" to abstain from subrequests and improve performance.')
+            ->end();
+
+        return $subTree;
+    }
+
+    /**
+     * @return ArrayNodeDefinition|NodeDefinition
+     */
+    private function getBackendConfig()
+    {
+        $tree = new TreeBuilder();
+        $subTree = $tree->root('backend');
+        $subTree->addDefaultsIfNotSet();
+        $subTree->children()
+            ->scalarNode('main_pagedef')
+                ->defaultValue('welcome')
+                ->info('The pagedef that is displayed after login and when clicking any "to main" button.')
```

Comment:
(for my understanding: this is then what is normally called "home page"?)

Reply:
Yes. I renamed it to home_pagedef to use the term "home" consistently.

Reply:
home_page_def?

Reply:
The `pagedef` notation is used throughout the code base, I wouldn't write it in another way here.